### ISSUE TITLE
weekly_start not responding in R

### ISSUE THREAD

**Issue by wimpel** [facebook/prophet#342]
R 3.4.2 , prophet v0.2 (installed from CRAN)
When I plot the components of the prophet

`prophet_plot_components(m, forecast, weekly_start = 2)`

I expect the chart with the weekly seasonality to start at "Monday".. However, this is not the case, it still starts at Sunday, no matter what value i pass to weekly_start.

**Comment by bletham** [comment]
Thanks, this was broken by the switch to datetimes - it is now adding seconds instead of days. I'll commit a fix ASAP, in the meantime you can get the expected behavior with `prophet_plot_components(m, forecast, weekly_start=2 * 60 * 60 * 24)`.

**Comment by bletham** [comment]
This is fixed in https://github.com/facebook/prophet/commit/af80f9c94e26528aaa7558479bbdff7a96ae1fd0. You can install it from github with
```
devtools::install_github('facebook/prophet', subdir='R', ref='master')
```

**Comment by bletham** [comment]
Fix pushed to CRAN in v0.2.1.
